doc/interfaces/reconstruction_flow.rst

```diff
@@ -0,0 +1,269 @@
+
+This command will save the CSD metrics to the specified output directory.
+
+----------------------------------
+Mean Apparent Propagator (MAP)-MRI
+----------------------------------
+
+The MAP-MRI basis allows for the computation of directional indices, such as
+the Return To the Axis Probability (RTAP), the Return To the Plane Probability
+(RTPP), and the parallel and perpendicular Non-Gaussianity.
+
+The estimation of analytical Orientation Distribution Function (ODF) and a
+variety of scalar indices from noisy DWIs requires that the fitting of the ]
+MAPMRI basis is regularized.
+
+
+------------------------
+Diffusion Tensor Imaging
+------------------------
+
+The diffusion tensor model is a model that describes the diffusion within a
+voxel. First proposed by Basser and colleagues [Basser1994]_, it has been very
+influential in demonstrating the utility of diffusion MRI in characterizing the
+micro-structure of white matter tissue and of the biophysical properties of
+tissue, inferred from local diffusion properties and it is still very commonly
+used.
+
+The diffusion tensor models the diffusion signal as:
+
+.. math::
+
+    \frac{S(\mathbf{g}, b)}{S_0} = e^{-b\mathbf{g}^T \mathbf{D} \mathbf{g}}
+
+Where $\mathbf{g}$ is a unit vector in 3 space indicating the direction of
+measurement and b are the parameters of measurement, such as the strength and
+duration of diffusion-weighting gradient. $S(\mathbf{g}, b)$ is the
+diffusion-weighted signal measured and $S_0$ is the signal conducted in a
+measurement with no diffusion weighting. $\mathbf{D}$ is a positive-definite
+quadratic form, which contains six free parameters to be fit. These six
+parameters are:
+
+.. math::
+
+   \mathbf{D} = \begin{pmatrix} D_{xx} & D_{xy} & D_{xz} \\
+                       D_{yx} & D_{yy} & D_{yz} \\
+                       D_{zx} & D_{zy} & D_{zz} \\ \end{pmatrix}
+
+This matrix is a variance/covariance matrix of the diffusivity along the three
+spatial dimensions. Note that we can assume that diffusivity has antipodal
+symmetry, so elements across the diagonal are equal. For example:
+$D_{xy} = D_{yx}$. This is why there are only 6 free parameters to estimate
+here.
+
+We will use the ``stanford_hardi`` dataset in DIPY to showcase this reconstruction
+method. As with any other workflow in DIPY, you can also use your own data!
+
+Create a directory in which to save the output volumes(e.g.:
+``recons_dti_output``)::
+    
+    mkdir recons_dti_output
+
+To run the Diffusion Tensor Imaging reconstruction method, we need to specify the 
+paths to the diffusion input file, b-values file, b-vectors file and mask file,
+followed by optional arguments. In this case, we will be specifying the list of
+metrics to save (``save_metrics``), the output directory (``out_dir``), and the
+name of the tensors volume to be saved (``out_tensor``).
+
+We will run the command as::
```

Comment:
The DTI reconstruction is performed by calling the {double_backticks}dipy_fit_dti{double_backticks} command, e.g.::

Reply:
Reworded in 4ce067c